draft-ietf-tls-tls13.md

```diff
@@ -3518,9 +3520,23 @@ other than for computing other secrets.)
 ##  MTI Cipher Suites
 
 In the absence of an application profile standard specifying otherwise, a
-TLS-compliant application MUST implement the cipher suite
-[TODO:Needs to be selected](https://github.com/tlswg/tls13-spec/issues/32).
-(See {{cipher-suites}} for the definition.)
+TLS-compliant application MUST implement the following cipher suites:
+
+~~~~
+    TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256 
+    TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
```

Comment:
A rather dumb nit: These are ordered "ECDSA, RSA" but the other pair is ordered "RSA, ECDSA". Could you please swap the order of one of these pairs? (I don't care which) It doesn't really affect anything; just looks odd to me. :p